Makefile

```diff
@@ -112,6 +112,14 @@ run: go.build
 
 .PHONY: cobertura reviewable manifests submodules fallthrough test-integration run crds.clean
 
+service: 
+	@if [ ! -d "./code-generator" ]; then \
+	 git clone "https://github.com/aws-controllers-k8s/code-generator.git"; \
+	fi
+	@cd ./code-generator && $(MAKE) 
```

Comment:
Do we need to run `make` in the code generator directory?

Reply:
We don`t need to run **make** in the code generator directory. The **make** will automatically run when the code-generator file is found.

Reply:
We simply have to run **make** in the AWS-provider main repo. After that, we have to write **make service <SERVICE_ID>=<some aws service name>**. That's it.

Reply:
Right, but it looks like this line is downloading the code generator then changing to its directory and running `make`?

Reply:
Yes , you are absolutely right.

Reply:
Sorry, I think I am not following correctly, what is the purpose of:
```
@cd ./code-generator && $(MAKE) 
```
in the `make service` target?

Reply:
It is going to invoke the `test` target in the code generator each time `make service ...` is run, which I am guessing we don't need to do here? https://github.com/aws-controllers-k8s/code-generator/blob/b977d86e11a017ea078437d80dd9f32bb11a3017/Makefile#L22